File: Challenges/day_5_if_else.py
#!/usr/local/bin/python

import sys

a = sys.argv[1]

if a == "1":
	print("a is one")
	print("This is still the then clause of the if statement")
else:
	print(f"a is {a}")
	print("this is still the else clause of the if statement")
print("This is after the if statement")
print(sys.argv)

File: challenges/day_5_if_else.go
package main

import (
	"fmt",
	"os"
)

func main() {
	var a int;
	if (a == "1"){
		fmt.Println("a is one");
		fmt.Println("This is still the then clause of the if statement");
	} else {
		fmt.Println("a is", a);
		fmt.Println("This is still the else clause of the if statement");
	}
	fmt.Println("This is after the if statement", os.Args);
}
